shaq_overflow/spec/models/question_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe Question, type: :model do
+ # pending "add some examples to (or delete) #{__FILE__}"
+end
+
+describe Question do
+  let (:user) {User.create(username: 'jay', email: 'jay@j.com', password: 'abc')}
+  let (:question) {Question.create(user_id: user.id, url: "http://www.w.com", title: "Title")}
```

Comment:
It's worth using create! rather than create in tests. That way you see any errors that happen. create returns false if it fails whereas create! throws an error